vestapol/typing/resources.py

```diff
@@ -0,0 +1,6 @@
+from vestapol.web_resources.csv_resource import CSVResource
+from vestapol.web_resources.github_resource import GitHubResource
+from vestapol.web_resources.json_resource import JSONResource
+from typing import Union
+
+ResourceTypes = Union[CSVResource, GitHubResource, JSONResource]
```

Comment:
Alternately, is there a way to tell mypy that anything of type `ResourceTypes` must be a subclass of `BaseResource`? If so, then we may not need to hard code a list of resource types.

Reply:
i was struggling with this for a while and couldn't figure it out so please lmk if you can find something else! I thought that [typing.Type](https://docs.python.org/3/library/typing.html#typing.Type) would work and it mostly did but I ran into issues when type checking the writer functions.